### PR TITLE
Docs: Add author configuration for blog postings.

### PR DESCRIPTION
**Pull request by pocmo**

### CODE REVIEW

File: docs/_data/authors.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  name: Christian Sadilek
+  image: https://avatars0.githubusercontent.com/u/472523?s=460&v=4
+  twitter: https://twitter.com/csadilek
+sebastian:
+  name: Sebastian Kaspari
+  image: https://avatars0.githubusercontent.com/u/89638?s=88&v=4
+  twitter: https://twitter.com/Anti_Hype
+fernando:
+  name: Fernando García Álvarez
+  image: https://avatars3.githubusercontent.com/u/4409856?s=400&v=4
```
Comment (pocmo) [inline docs/_data/authors.yml]: @fercarcedo Is this okay? Do you have a Twitter handle you want to use? (It's optional)

Comment (pocmo) [inline docs/_data/authors.yml]: https://avatars0.githubusercontent.com/u/472523?s=460&v=4

Comment (fercarcedo) [reply docs/_data/authors.yml]: Yes, that's okay :) My Twitter handle is @fergaral96

Comment (pocmo) [reply docs/_data/authors.yml]: Okay, added. :)

**Pull request merged**